### ISSUE TITLE
[Linux] Build fails with Nodejs >= v10

### ISSUE THREAD

**Issue by sum01** [RocketChat/Rocket.Chat.Electron#726]
Building from source using [this PKGBUILD](https://aur.archlinux.org/packages/rocketchat-desktop) on Arch Linux no longer works.

## My Setup
- Operating System: Arch Linux
- App Version: 2.10.5
I also tried modifying the PKGBUILD to build on 50922b10bfa5daafc48e9ed0ddba63859c256dd9 but the same error occurs.

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description

Attempt to build using the [PKGBUILD](https://aur.archlinux.org/packages/rocketchat-desktop) which previously worked. I assume a newer version of something broke the build. See [this issue with the same error](https://github.com/nodejs/node/issues/20281).

### Current Behavior

Build log from `makepkg -L`
```
yarn install v1.6.0
[1/4] Resolving packages...
[2/4] Fetching packages...
info 7zip-bin-mac@1.0.1: The platform "linux" is incompatible with this module.
info "7zip-bin-mac@1.0.1" is an optional dependency and failed compatibility check. Excluding it from installation.
info 7zip-bin-win@2.1.1: The platform "linux" is incompatible with this module.
info "7zip-bin-win@2.1.1" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.1.3: The platform "linux" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
info node-mac-notifier@0.0.13: The platform "linux" is incompatible with this module.
info "node-mac-notifier@0.0.13" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
$ electron-builder install-app-deps
(node:9586) ExperimentalWarning: The fs.promises API is experimental
  • electron-builder version=19.56.0
  • loaded configuration file=package.json ("build" field)
  • rebuilding native production dependencies platform=linux arch=x64
  • rebuilding native dependency name=spellchecker
  • rebuilding native dependency name=@paulcbetts/system-idle-time
Done in 20.28s.
yarn run v1.6.0
$ gulp build --env=production
gulp[10114]: ../src/node_contextify.cc:628:static void node::contextify::ContextifyScript::New(const v8::FunctionCallbackInfo<v8::Value>&): Assertion `args[1]->IsString()' failed.
 1: node::Abort() [gulp]
 2: 0x5620f098829b [gulp]
 3: node::contextify::ContextifyScript::New(v8::FunctionCallbackInfo<v8::Value> const&) [gulp]
 4: v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo*) [gulp]
 5: 0x5620f0b92e97 [gulp]
 6: 0x5620f0b9368f [gulp]
 7: 0x13523d20427d
error Command failed with signal "SIGABRT".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[1m[31m==> ERROR:(B[m[1m A failure occurred in build().(B[m
[1m    Aborting...(B[m
```

### Expected Behavior

That it builds like it did a few months ago (2018-02-07).

**Comment by sum01** [comment]
So it seems that something with Nodejs v10 (or v9?) is causing issues. This builds correctly by using `nodejs-lts-carbon`, which provides nodejs v8.11.2.

**Comment by sum01** [comment]
Fixed in v2.11.0
